.travis.yml

```diff
@@ -0,0 +1,6 @@
+language: node_js
+node_js:
+  - "6"
```

Comment:
Try it out -- open a new PR and see how it does! But can't you also do both? https://github.com/publiclab/PublicLab.Editor/blob/master/.travis.yml#L3-L4